### ISSUE TITLE
The top-K inhibitory gate may be hurting individuation

### ISSUE THREAD

**Issue by lordbasilaiassistant-sudo** [lordbasilaiassistant-sudo/BetterThanLLM#4]
A config search against `experiments/fitness/gauntlet.py` turned up something worth chasing: **the best configuration found so far switches the top-K gate off.**

### The finding

`README.md` describes action selection as:

> reactive (W_action + disposition) + planning (W_trans @ intention) + top-K gate

with `top_k = 2` as a "global inhibitory top-K gate" — presented as part of the architecture. But an 80-candidate random search, validated on a **disjoint** seed family, produced:

| config | search | holdout | chance floor (holdout) |
|---|---|---|---|
| baseline (`top_k=2`, `plan_weight=0.6`) | +0.5778 | +0.6129 | 0.6927 |
| **cand047** (`top_k=3`, `plan_weight=1.97`) | +0.8013 | **+0.7652** | **0.7155** |

`+0.1523` on seeds it never saw. `top_k=3` with `N_ACTIONS=3` means the gate does nothing — `softmax_topk` falls through to a plain softmax.

Full config: `lr=0.0404, disp_lr=0.0021, intent_lr=0.0174, plan_weight=1.97, top_k=3, noise=0.082, n_replay=400`.

### Why I think it's real and not another metric artifact

The obvious worry is that it won by moving the metric's floor — that is exactly how the *previous* search winner cheated (see #3). It didn't: cand047's chance floor is **higher** (0.7155 vs 0.6927), so it wins against a *harder* metric, not an easier one. Its overfitting gap is also small (+0.036).

Breakdown vs baseline: individuation **+0.9567** (vs +0.5908), cross_world **+0.6397** (vs +0.4680), autobiography +0.6992 (vs +0.7799 — slightly *worse*).

### The open questions

1. **Is the gate load-bearing at all?** It is described as architecture but no experiment isolates it. `plan_weight` also tripled here, so the two are confounded — a clean 2-D sweep over `top_k × plan_weight` would separate them.
2. **Does the gate trade individuation for autobiography?** The one measure that got worse is the non-cosine one. That could be a real trade-off rather than noise.
3. **Does this survive a bigger search and more seeds?** n=80 candidates, 8 seeds. It should be re-run wider before anyone edits the defaults.

### What would settle it

```bash
python3 experiments/fitness/search.py 300
```

plus a targeted sweep holding everything but `top_k` fixed. **Please don't change the published defaults on the strength of one search run** — the point of this issue is that it needs a proper answer first.
